apps/frontend/src/app/Components/Artifact/ArtifactFilterDisplay.tsx

```diff
@@ -95,6 +115,14 @@ export default function ArtifactFilterDisplay({ filterOption, filterOptionDispat
     })

```

Comment:
Pre-existing, but could you fix the warnings for `artSubTotal`?
![image](https://user-images.githubusercontent.com/36019388/213948382-c6b1c658-ae4f-49a9-b7d3-ad86d4e94a8c.png)